src/agent/src/rpc.rs

```diff
@@ -584,24 +584,31 @@ impl AgentService {
         let cid = req.container_id;
         let eid = req.exec_id;
 
-        let writer = {
+        let mut resp = WriteStreamResponse::new();
+        resp.set_len(req.data.len() as u32);
+
+        // EOF of stdin
+        if req.data.len() == 0 {
```

Comment:
Some CIs report error that `req.data.len()== 0` should be changed to `req.is_empty()` to make CIs happy.

Reply:
sorry, req.is_empty() -> req.data.is_empty()

Reply:
done, thanks